src/app/page.tsx

```diff
@@ -91,6 +91,7 @@ async function Home() {
 
   return (
     <div className="flex flex-col">
+      <SubscribeDialogLauncher />
```

Comment:
This is a server component so I needed to add the client behavior dialog as it's own component